README.md

```diff
@@ -1,50 +1,84 @@
+-   [UX Design on Figma](https://www.figma.com/file/JC406azeeeGGlTbLAxCs04/Feeling-Tracker?node-id=0%3A1&t=jyrThFgzn0f7LWof-0)
 
 ## Origin Story
 
-Due to the worsening of my chronic medical conditions, I had to start tracking my daily activity. 
+My chronic health conditions got worse, so I needed to track my daily activities closely. I couldn't find any existing apps that met my specific privacy and functionality needs, so I created my own. After sharing with friends, I saw a greater need and chose to release it as an open-source app for everyone.
+
+## Features
+
+-   Activity Management: Create, modify, and delete activities with ease.
+-   Category Management: Define, edit, and organize categories for activities.
+-   Organize with Categories: Assign activities to specific categories for enhanced tracking.
+-   Quick Duplication: Clone activities to quickly repeat them without re-entry.
+-   Activity Templates: Save activities as templates for future use.
+-   Data Handling: Options to import, export, and permanently delete user data.
+-   Daily Overview: Access a daily view of activities.
+-   Navigate Weekly: Browse through days with a convenient weekly carousel.
+
+<br>
+<a href="https://imgur.com/7r0Vrju"><img src="https://i.imgur.com/7r0Vrju.gif" title="source: imgur.com" /></a>
 
-After thoroughly searching the Google Play Store, none of the available apps were good enough (or private enough) to suit my very specific needs.
+## How to Contribute
 
-I decided to make my own app to track my daily activities, with the ability to trace back weeks or even months and see useful trends.
+Thank you for your interest in contributing to our project! Here's how you can help:
 
-Talking to some friends, I realized there's more demand for such app than I had imagined, so I decided to make it a public open-source project.
+### Reporting Issues
 
-## Tech Stack
+Found a bug or have a feature suggestion? Please check to see if it has already been reported or suggested in the issues tab. If not, go ahead and open a new issue, providing as much relevant information as possible to help us understand the request or problem.
 
-The app is built using Vite, React, TypeScript and Dexie.
+### Contributing Code
 
-### Styles:
-This app uses SCSS and follows [Elad Shechter](https://eladsc.com/)'s [Storytelling Architecture](https://eladsc.com/2019/11/29/css-story-architecture-talk/) (at least for the folder hierarchy part) to keep the files neatly organized. 
+We are happy to accept pull requests, If you'd like to contribute to the codebase, here's a quick guide:
 
-### PWA and Local Storage:
-To maximize privacy on this app, all of the user's data will be stored in local storage via [Dexie](https://dexie.org) (an IndexedDB wrapper). No server needed! Just "install" it locally and you're good to go.
+1. Fork the repository and create your branch from `main`.
```

Comment:
Maybe they should fork from `development` as it should always be more up-to-date than `main` which is used for production